shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/main/presentation/MainComponent.kt

```diff
@@ -108,6 +108,7 @@ class DefaultMainComponent(
                 PlantDetailComponent(
                     plant = config.plant,
                     componentContext =  context,
+                    appModule = appModule,
```

Comment:
I think it would be better to use 'appModule.shareUtils' instead of 'appModule' here :)

Reply:
yep you are right